Langs/0410.lang

```diff
@@ -0,0 +1,59 @@
+[MetaData]
+Language=Italian - Italy
+Translator=Mario (@MarioMarietto)
+Version=2.2.5.0
+[Strings]
+Donate=Dona
+Check for Updates=Verifica la disponibilità di aggiornamenti
+Your Windows 11 Compatibility Results are Below=I risultati della compatibilità con Windows 11 sono mostrati sotto
+Now Reach WhyNotWin11 via https://www.whynotwin11.org/=Raggiungi WhyNotWin11 attraverso https://www.whynotwin11.org/
+Results Based on Currently Known Requirements!=Risultati basati sui requisiti di sistema attualmente conosciuti!
+Translation by=Traduzione di
+Architecture (CPU + OS)=Architettura (CPU + SO)
+Boot Method=Metodo Boot
+CPU Compatibility=Compatibilità CPU
+CPU Core Count=Core CPU
+CPU Frequency=Frequenza CPU
+DirectX + WDDM2=DirectX + WDDM2
+Disk Partition Type=Tipo partizione del disco
+RAM Installed=RAM Installata
+Secure Boot=Secure Boot
+Storage Available=Spazio di archiviazione disponibile
+TPM Version=Versione TPM
+Checking...=Verifica in corso...
+64 Bit CPU=CPU a 64 Bit
+64 Bit OS=SO a 64 Bit
+32 Bit CPU=CPU a 32 Bit
+32 Bit OS=SO a 32 Bit
+Unable to Determine=Impossibile determinare
+Unable to Check List=Impossibile verificare la lista
+Error Accessing List=Errore durante l'accesso della lista
+Not Currently Listed as Compatible=Attualmente non elencato come compatibile
+Listed as Compatible=Elencato come compatibile
+Cores=Core
+Threads=Thread
+GPT Detected=GPT individuato
+GPT Not Detected=GPT non individuato
+Enabled=Abilitato
+Supported=Supportato
+Not Supported=Non supportato
+Disabled / Not Detected=Disabilitato / Non individuato
+Drive(s) Meet Requirements=La/le unità soddisfa/soddisfano i requisiti
+TPM Missing / Disabled=TPM Mancante / Disabilitato
+Detected=Individuato
+DirectX 12 and WDDM 3=DirectX 12 e WDDM 3
+DirectX 12 and WDDM 2=DirectX 12 e WDDM 2
+No DirectX 12, but WDDM2=DirectX 12 mancante ma WDDM 2 presente
+DirectX 12, but no WDDM2=DirectX 12 presente ma WDDM 2 mancante
+No DirectX 12 or WDDM2=DirectX 12 e WDDM 2 mancanti
+Test Build?=Verificare la build?
```

Comment:
By test built is meant testing/prerelease, is this in the right context?

Reply:
Ahhh alright no sorry. It should be: "Build di test?" then. I've made another commit to fix this issue :)